lib/otp/jwt/token.rb

```diff
@@ -37,6 +37,9 @@ def self.sign(payload)
       #
       # @return [Hash], JWT token payload
       def self.verify(token, opts = nil)
+        if self.jwt_algorithm == 'none'
+          return ::JWT.decode(token.to_s, nil, false, opts || {})
+        end
```

Comment:
@timle8n1 thank you for this! It totally makes sens.

May I suggest a more explicit approach to your changeset (It took me a bit to understand what that magical true/false param is about :see_no_evil:)... Please consider this instead:
```
verify = true
verify = false if self.jwt_algorithm == 'none'
::JWT.decode(token.to_s, self.jwt_signature_key, verify, opts || {})
```

We should probably link in the docs to the relevant `JWT` note on how to use the `none` algorithm. What do you think?

Thank you!!! :bow: :heart:

Reply:
Sure thing - let me know if that update works for you.